pytorch_lightning/utilities/exceptions.py

```diff
@@ -23,3 +23,9 @@ class DeadlockDetectedException(Exception):
     """
     Exception used when a deadlock has been detected and processes are being killed
     """
+
+
+class DummyException(Exception):
```

Comment:
I think this should not be in utilities and only in the test directory, for example `tests/helpers/` because testing code is not included in the distribution.